From b68f2944099d1dae501e863b29921337fa1381c9 Mon Sep 17 00:00:00 2001
From: "foo@vm" <foobar@cc>
Date: Tue, 5 May 2015 13:16:54 +0800
Subject: [PATCH] fix gitignore problem

---
 .gitignore | 1 +
 1 file changed, 1 insertion(+)

diff --git a/.gitignore b/.gitignore
index 1f3f1a9..7e3cd94 100644
--- a/.gitignore
+++ b/.gitignore
@@ -46,6 +46,7 @@ compile_commands.json
 projects/*
 !projects/CMakeLists.txt
 !projects/Makefile
+!projects/LLVMBuild.txt
 # Clang, which is tracked independently.
 tools/clang
 # LLDB, which is tracked independently.
-- 
1.9.1

