From 1807993b4837bddd73550d69fe25cad0bfe785f8 Mon Sep 17 00:00:00 2001
From: ganboing <ganboing@gmail.com>
Date: Tue, 5 May 2015 22:35:17 +0800
Subject: [PATCH] update gitignore for Cpu0

---
 .gitignore | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/.gitignore b/.gitignore
index 7e3cd94..238f895 100644
--- a/.gitignore
+++ b/.gitignore
@@ -59,6 +59,8 @@ tools/llgo
 tools/polly
 # Sphinx build tree, if building in-source dir.
 docs/_build
+# Cpu0 backend
+lib/Target/Cpu0
 
 #==============================================================================#
 # Files created in tree by the Go bindings.
-- 
1.9.1

